File: 1-Plantilla-procesamiento-de-datos/data_preprocesing_template.py
# -*- coding: utf-8 -*-
"""
Created on Sun Aug 30 22:41:46 2020

@author: Facundo
"""


#Plantilla standard


#Plantilla de Preprocesado

#Importar librerias

import numpy as np
import matplotlib.pyplot as plt
import pandas as pd

# Importar data set
dataset = pd.read_csv("Data.csv")

X =dataset.iloc[:, :-1].values 
###X = dataset.iloc[filas que quiero , columnas que quiero].value
y =dataset.iloc[:, 3].values

#Dividir el Dataet en  conjunto de entrenamiento y conjunto de testing

from sklearn.model_selection import train_test_split
X_train, X_test, y_train, y_test = train_test_split(X, y, test_size=(0.2), random_state=0) #random state es la semilla


#Escalado de Variable
 #from sklearn import preprocessing  
"""#---- Esto es a veces pero es de uso 
sc_X = preprocessing.StandardScaler()
X_train = sc_X.fit_transform(X_train)
X_test = sc_X.transform(X_test)"""

File: 1-Plantilla-procesamiento-de-datos/data_preprocesing_template.R
#Planilla para el procesamiento de datos

#importar el dataset

dataset = read.csv('Data.csv')
#dataset = dataset[,2:3] Esto es para solo usar columnas 2 y 3 del data set

#Dividir los datos en conjunto de entrenamiento y conjunto de test
library(caTools)
set.seed(123)
split = sample.split(dataset$Purchased, SplitRatio = 0.8)
training_set= subset(dataset, split==TRUE)
testing_set= subset(dataset, split==FALSE)

#Escalado de Valores
#training_set[,2:3] = scale(training_set[,2:3])
#testing_set[,2:3] = scale(testing_set[,2:3])